### PR TITLE
General 80 full chores

### PR DESCRIPTION
**Pull request by bathientran**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Include the motivation and context, and list any dependencies that are required for this change.

Adds the chores detail page.

## Link to Ticket

Please indicate the ticket(s) in Notion that correspond to this PR.

- https://www.notion.so/bathientran/Full-Chore-List-View-ed8e04b4a2454c15959bd4cf3ab1e0f5?pvs=4

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## What Has Changed

Please describe in bullets what you have changed.

- Changes API routing to consolidate utils.
- Adds room members endpoint.
- Adds Image usage.
- Adds standardized color usage.
- Adds chore detail page with filtering for chores, updating chores, viewing chores, adding chores, selecting users to assign.

## How Has This Been Tested / How Can This Be Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

https://www.loom.com/share/6602ea79c7c84e938cfcad34bf42abb4?sid=7bd31719-f61d-4be7-b7ed-a965f1556e5a

### CODE REVIEW

File: assets/seed-images/user_image_1.jpg
Comment (vickipetrova) [inline assets/seed-images/user_image_1.jpg:1]: aren't the assets added by the previous PR for the profile creation?

**Review by vickipetrova — CHANGES_REQUESTED** [review]
I think this PR also includes the changes from the profile creation PR https://github.com/minerva-startup-spring2024/roommate-organizer/pull/23/ 

We should probably separate them?

File: assets/seed-images/user_image_1.jpg
Comment (bathientran) [reply assets/seed-images/user_image_1.jpg:1]: Oh I set the base incorrect, let me change that

**Comment by vickipetrova** [comment]
image

**Comment by bathientran** [comment]
> image

does npm install fix it? should just be an install

**Comment by vickipetrova** [comment]
image
image
https://github.com/advisories/GHSA-hhhv-q57g-882q

**Review by vickipetrova — APPROVED** [review]

**Comment by vickipetrova** [comment]
> > image
> 
> does npm install fix it? should just be an install

i can't see the image, i tried again but i cant get past the login page (i sent the bug in telegram), idk if it happens just locally for me or if it's the same for other people -- https://t.me/c/2090124433/28/452

**Pull request merged**